src/app/components/PageTitleConfirmation.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type FunctionWithProperty = {
+  (title: string): void;
+  fn?: (title: string) => void;
+};
+
+const announcePageTitle: FunctionWithProperty = (title: string) => {
```

Comment:
i think this component should work based on being connected to the redux state instead of having a separate callback like this